accounts/views.py

```diff
@@ -20,7 +24,18 @@ def post(self, request, *args, **kwargs):
             "user": UserSerializer(user, context=self.get_serializer_context()).data
         })
 
-   # Edit a User
+    
+    @staticmethod
+    @receiver(signals.post_save, sender=my_user)
+    def user_save_observer(sender, instance, created, **kwargs):
+        if(created):
```

Comment:
so on user creation, we're creating a health profile?

Reply:
we could do this while we're creating the user object

Reply:
Yeah that was the issue that I talked about yesterday...... And I am initializing it with default values as we are going to give option to user to edit it later through UI......
Also I though it would be better handled with signals so I implemented it this way...